### ISSUE TITLE
About News Dataset

### ISSUE THREAD

**Issue by JiamingUWU** [dreasysnail/POINTER#8]
In your paper, The EMNLP2017 WMT News dataset5 contains 268,586 sentences, but there are lots of datasets in url http://www.statmt.org/wmt17/ and I have no sense which one is the dataset used in experiments. I'd be appreciated if you provide some details.

**Comment by guoyinwang** [comment]
We use the news data obtained from https://github.com/pclucas14/GansFallingShort/tree/master/real_data_experiments/data/news

**Comment by JiamingUWU** [comment]
sorry to disturb you. How is the data set divided into training set , dev set and test set?
